tests/e2e/standalone/commands.go

```diff
@@ -125,20 +124,18 @@ func cmdStop(appId string, args ...string) (string, error) {
 	return spawn.Command(common.GetDaprPath(), stopArgs...)
 }
 
-// cmdUninstall uninstalls Dapr with --all flag and returns the command output and error.
-func cmdUninstall(args ...string) (string, error) {
+// cmdUninstallAll uninstalls Dapr with --all flag and returns the command output and error.
+func cmdUninstallAll(args ...string) (string, error) {
 	uninstallArgs := []string{"uninstall", "--all"}
-
-	daprContainerRuntime := containerRuntime()
-
-	// Add --container-runtime flag only if daprContainerRuntime is not empty, or overridden via args.
-	// This is only valid for non-slim mode.
-	if !isSlimMode() && daprContainerRuntime != "" && !utils.Contains(args, "--container-runtime") {
-		uninstallArgs = append(uninstallArgs, "--container-runtime", daprContainerRuntime)
-	}
```

Comment:
Why did we remove this instead of moving it to `cmdUninstall` and using this instead of the new function in `utils`?